caffe2/python/tutorials/py_gen/MNIST.py

```diff
@@ -128,20 +152,52 @@ def AddInput(model, batch_size, db, db_type):
     return data, label
 
 
-# At this point we need to take a look at the predictions coming out of the network at convert them into probabilities.
-# "What's the probability that this number we're looking at is a 5", or "is this a 7", and so forth. 
+# Now we are going to construct our own model. In this part input is our data blob. The output is a blob which will contain probabilities for each of the 10 possible digits.
+# 
+# We are going to use so called Multilayer Perceptron architecture (MLP). Relu activation function is going to be used:
+# 
+# Relu(x) = x if x > 0 else 0
+# 
+# Each layer of an MLP is just matrix multiplication with a bias + an activation function. I.e.:
```

Comment:
Each layer of an MLP is just matrix multiplication with a bias plus an activation function. In our case, with ReLU activation, that is: